content/en/docs/_index.md

```diff
@@ -8,3 +8,71 @@ menu:
 aliases:
   - /documentation
 ---
+
+Jenkins X is an open source project that provides Kubernetes-based CI/CD
+applications featuring pipeline automation, built-in GitOps, and preview
+environments to help teams collaborate and accelerate their at any scale. [Read
+More...](/docs/overview/)
+
+## Get started with Jenkins X in 3 easy steps!
+
+
+### Step 1: Install the Jenkins X Binary
+
+The first step in using Jenkins X is installing the Jenkins X binary (`jx`) for
+your operating system.
+
+<div style="text-align:center">
+<a class="btn btn-lg btn-secondary mr-3 mb-4" href="/docs/getting-started/setup/install/#linux">
+		Install Linux <i class="fab fa-linux ml-2 "></i>
+    </a> 
+<a class="btn btn-lg btn-dark mr-3 mb-4" href="/docs/getting-started/setup/install/#macos">
+		Install macOS <i class="fab fa-apple ml-2 "></i>
+	</a>
+</div>
+
+[More installation options... ](/docs/getting-started/setup/install/)
+
+### Step 2: Provision your Kubernetes Cluster
+
+Once you have installed the binary on your workstation, next you want to provision your kubernetes cluster.  Choose your cloud provider.
+
+<div class="row">
```

Comment:
Do we really need so much custom HTML? Have we explored all options to get this decently laid out and rendered with Markdown and a minimal amount of HTML?